### ISSUE TITLE
Create JSFiddle does not add resources to the created fiddle

### ISSUE THREAD

**Issue by petrsloup** [openlayers/ol3#4121]
Some examples have external resources specified, such as:
```
resources:
  - http://cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.6/proj4.js
```

When creating the JSFiddle using the "Edit" button, these resources are not passed to the fiddle and it does not work properly.
According to http://doc.jsfiddle.net/api/post.html, it should be quite easy to pass the urls.

**Comment by marcjansen** [comment]
Will you be able and qilling to create a PR?

**Comment by bartvde** [comment]
I'm quite sure this used to work

**Comment by ahocevar** [comment]
No @bartvde, this never worked.

**Comment by petrsloup** [comment]
Looks like @ahocevar fixed this with #4267
